back/spec/acceptance/admin_publications_spec.rb

```diff
@@ -737,7 +737,7 @@
         parameter :remove_not_allowed_parents, 'Filter out folders which contain only projects that are not visible to the user', required: false
         parameter :include_publications, 'Include the related publications and associated items', required: false
 
-        example_request 'Index action does invoke unnecessary queries' do
+        example_request 'Index action does not invoke unnecessary queries' do
```

Comment:
Can't figure out how many projects are available in this test. There are quite a lot of queries, but because this is a controller, it's hard to know which ones are related to the actual endpoint and which are just rails setup. Maybe another reason to put in a service?